packages/fields/src/fields/index.ts

```diff
@@ -1,5 +1,6 @@
 export { default as slugField } from './slug';
 export { default as titleField } from './title';
+export { default as templateTitleField } from './template-title';
```

Comment:
Why is this not just the "titleField"? What's different?

Reply:
Different `label`, some flags like `enableGlobalSearch` and different `render`. For `render` though if we create a separate `pageTitleField` as I suggest, it will be the same with the basic title field:
>Create new pageTitleField which would look like the current title field in trunk. page is the only post type that cares about front page and posts page



When we'll want to do that for patterns too, it's going to have a different `getValue` too.

Reply:
To be honest, I feel like these should be unified, the differences seems a bit random and there's no clear user argument for it. but I agree that it might not be the right PR for it.

Reply:
For me it's not about the flags like `enableGlobalSearch` and not even about the `getValue`, as I don't think it hurts to add an extra check for finding the proper title in some other props too (for patterns). Not even the `label` as we could rename `template` to `title`.

It's mostly about the `render`. For example why have the store subscriptions for `frontpage and posts page` in any other post type except `page`? By looking our current core types (patterns, templates, any post type, pages) I think it's fine to have separate fields that use the underlying `render` function (`BaseTitleView`).

I'm saying all these because I'd like to also include the `pageTitleField` and `patternTitleField` in this PR and not create semi-abstractions that feel like proper ones (current titleField in trunk) but don't work well with the other post types.

What do you think?

Reply:
These are the differences I see:

- post => render the `BaseTitleView`
- templates => render the `BaseTitleView` and enable global search (the only entity that needs it)
- pages => render `PagesTitleView` (differs from `BaseView` because it queries to site)
- patterns => render `PatternsTitleView` (with the logic to visualize highlight sync/unsync, etc.)

There're other details that could be aligned (label).

If we aim to display those fields in the same way across the app, yeah, we should have one title field for each entity. Following this logic, `titleField` should be renamed `pageTitleField`.

Reply:
I think "front-page" and "posts-page" conceptually shouldn't be part of the title field.

Reply:
It seems to me we agree on having these four fields, right? If yes, I can proceed with these changes.

Reply:
I agree for now :) given the design constraints. I still think we should aim to have one field for all the CPTs that support "title" and find different ways to address these design needs.